examples/catalog/bin/sample_page.dart

```diff
@@ -105,27 +115,29 @@ class SampleInfo {
     final RegExp startRE = RegExp(r'^/\*\s+^Sample\s+Catalog', multiLine: true);
     final RegExp endRE = RegExp(r'^\*/', multiLine: true);
     final Match startMatch = startRE.firstMatch(contents);
-    if (startMatch == null)
-      return false;
+    if (startMatch == null) return false;
 
     final int startIndex = startMatch.end;
     final Match endMatch = endRE.firstMatch(contents.substring(startIndex));
-    if (endMatch == null)
-      return false;
+    if (endMatch == null) return false;
```

Comment:
Not an improvement

Reply:
ack. Switching the style to match the original to shrink the diff.